### DISCUSSION TITLE
Best Host for Binance

### DISCUSSION THREAD

**Discussion by nithi2023** [ctubio/Krypto-trading-bot#1091] [Q&A] [answered]
I am currently on OVH FR and connecting Binance. When I run latency test, results are around `1100-1800ms` and K advised me to move to another host. I tried AWS SG-1 (free tier) and the results are almost same. Any suggestion how to choose best host provider which is best for binance  ?

**Comment by ctubio** [comment, marked as answer]
:koala: Binance API is supposed to be on AWS Tokyo and New Jersey

**Reply by nithi2023** [reply]
@ctubio Thanks buddy!

**Comment by adamantmm** [comment]
1100-1800ms is almost certainly not exchange RTT — that sounds like a bad route or a latency test that includes DNS/TLS/connect overhead on every sample.

For Binance spot/futures API work, Tokyo (ap-northeast-1) is the usual starting point. But measure properly: persistent HTTP keep-alive or WS, 100+ samples, report p50/p95. A single curl per test is misleading.

Also check whether your provider routes EU→Asia through the US. OVH FR and AWS SG can both look fine on paper and still give you 1s+ if the path is bad. A small VPS in Tokyo with mtr to api.binance.com is worth 10 minutes before moving hosts again.
